experimental/framework/tree-react-api/src/useTree.ts

```diff
@@ -19,6 +21,8 @@ import React from "react";
  * Use deep clone if saving a copy for comparison.
  *
  * Not currently compatible with 'React.memo'.
+ *
+ * @deprecated Prefer using the new TypedTree entry point and tree types, and the hooks for them, {@link useTreeContext} and {@link useSubtree}.
```

Comment:
FWIW, I'm pretty sure we copied this into Nick's demo.  You should be able to delete/replace without causing anyone grief.